Sources/AppwriteModels/Execution.swift

```diff
@@ -52,6 +52,9 @@ public class Execution {
     /// Function execution duration in seconds.
     public let duration: Double
 
+    /// The scheduled time for execution. If left empty, execution will be queued immediately.
+    public let scheduledAt: String??
```

Comment:
We have a double-optional here, but it looks like this is not a new issue. Will create a quick fix PR

Reply:
Thanks Jake 👍